.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
`.vscode` folder shouldn't be commited, since thisare user-specific preferences and not project-specific. More info https://blog.martinhujer.cz/dont-put-idea-vscode-directories-to-projects-gitignore/

Reply:
Deleted